conf/app.routes

```diff
@@ -20,3 +20,8 @@ GET        /status/closed                               controllers.EstateStatus
 GET        /status/problem-with-service                 controllers.EstateStatusController.problemWithService()
 GET        /status/utr-does-not-match-records           controllers.EstateStatusController.utrDoesNotMatchRecords()
 GET        /status/account-not-linked                   controllers.EstateStatusController.accountNotLinked()
+
+GET        /UTR                                         controllers.UTRController.onPageLoad(mode: Mode = NormalMode)
```

Comment:
can this be /unique-tax-reference-number

Reply:
Updated